overlord/snapstate/autorefresh.go

```diff
@@ -564,7 +564,7 @@ func (m *autoRefresh) launchAutoRefresh(overrideDelay bool) error {
 	}()
 
 	// NOTE: this will unlock and re-lock state for network ops
-	updated, updateTss, err := AutoRefresh(auth.EnsureContextTODO(), m.state)
+	updated, updateTss, err := AutoRefresh(auth.EnsureContextTODO(), m.state, &AutoRefreshOptions{IsContinuedAutoRefresh: overrideDelay})
```

Comment:
do we have a unit test for this change?

Reply:
I added a test now that checks this but not sure if it's the best way (it's a bit indirect), maybe @MiguelPires has a good idea how to make this more direct.

Reply:
Looks fine